### ISSUE TITLE
Trimming not working with custom materials on server

### ISSUE THREAD

**Issue by DeepRsnanceX** [Hipposgrumm/armor-trims#35]
Hey, i'm having an issue where i'm not being able to use the custom materials i've defined in the config with any armor at all. The materials ARE recognized, since they show up in the shift menu of available ingredients for trims, and when using them in the smithing template i can see the design with the proper color, but i can't take the resulting item from the slot and the arrow has an X on it.

Here's an example:

![image](https://github.com/Hipposgrumm/armor-trims/assets/119914418/bd18f150-45d6-46a3-82ee-c4dad3418714)
Available ingredients list, highlighted the ones i'm using here

![image](https://github.com/Hipposgrumm/armor-trims/assets/119914418/a0122f12-763e-465a-bc51-35f10bc0fc5e)
Trimming with diamond, works fine

![image](https://github.com/Hipposgrumm/armor-trims/assets/119914418/8a5cc703-277c-41f9-933e-65ed13d09c3f)
Trimming with Falsite, doesn't work

Is there anything that could be causing it? or any potential fix i can do?

**Comment by Hipposgrumm** [comment]
~~Maybe it doesn't work with Falsite because Falsite is false.~~
Jokes aside, have you restarted the game after adjusting the config setting?
*~~Also this report may become invalid because I am changing from materials to tags.~~*

**Comment by fibberclip** [comment]
yeah i did, this is happening in a server i run and both server and client were restarted after changing the configs, both server and client have the same configs as well, with all of the materials listed in both sides.
Trying this in singleplayer DOES work just fine from my testing, which is why i thought it might've been a conflict with the configs in server/client sides, but doesn't seem like it

**Comment by Hipposgrumm** [comment]
Again, this will hopefully get fixed with the transition ot tags, but it's something to test.

**Comment by fibberclip** [comment]
I'll take any fix honestly lol, do u have any estimate of when the next update might release?

**Comment by Hipposgrumm** [comment]
I don't know. It's been quite a while.

**Comment by Hipposgrumm** [comment]
Actually I'm gonna reopen this since it's not out yet.
